packages/components/package.json

```diff
@@ -56,35 +56,35 @@
   "devDependencies": {
     "@chanzuckerberg/axe-storybook-testing": "^5.0.0",
     "@chanzuckerberg/story-utils": "^3.0.4",
-    "@storybook/addon-a11y": "^6.4.13",
-    "@storybook/addon-essentials": "^6.4.13",
-    "@storybook/addon-interactions": "^6.4.14",
+    "@storybook/addon-a11y": "^6.4.17",
+    "@storybook/addon-essentials": "^6.4.17",
+    "@storybook/addon-interactions": "^6.4.17",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/react": "^6.4.13",
+    "@storybook/react": "^6.4.17",
     "@storybook/storybook-deployer": "^2.8.10",
     "@storybook/testing-library": "^0.0.8",
     "@storybook/testing-react": "^1.2.3",
-    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
-    "@types/jest": "^27.0.2",
+    "@types/jest": "^27.4.0",
     "@types/react": "^17.0.16",
-    "autoprefixer": "^10.4.0",
+    "autoprefixer": "^10.4.2",
     "axe-core": "^4.3.5",
-    "chromatic": "^6.0.5",
+    "chromatic": "^6.4.3",
     "codecov": "^3.8.3",
     "flowgen": "^1.14.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.3.1",
-    "postcss": "^8.3.11",
-    "postcss-cli": "^9.0.1",
+    "jest": "^27.4.7",
+    "postcss": "^8.4.6",
+    "postcss-cli": "^9.1.0",
     "postcss-import": "^14.0.2",
     "postcss-nested": "^5.0.6",
     "react": "^16.14.0",
     "react-docgen-typescript-loader": "^3.7.2",
     "react-dom": "^16.14.0",
     "storybook-css-modules-preset": "^1.1.1",
-    "tailwindcss": "^2.2.7",
-    "ts-jest": "^27.0.7",
-    "typescript": "^4.5.4"
+    "tailwindcss": "^2.2.19",
```

Comment:
did you see issues with moving to v3? or just safer to be in a separate PR? we have this ticket but I haven't looked into the changelog much: https://app.shortcut.com/czi-edu/story/176262/upgrade-to-tailwind-3-0

Reply:
I didn't look at moving to v3. Wasn't sure if there were reasons to stay on the same major version as LP or not.

But I can't think of a reason why it would be problematic (since we ship compiled styles, not Tailwind classnames).